example/data/lib/payments/repository/network_payments_data_repository.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:data/payments/api/payments_data_api_client.dart';
+import 'package:domain/payments/model/creditor.dart';
+import 'package:domain/payments/repository/payments_data_repository.dart';
+
+class NetworkPaymentsDataRepository extends PaymentsDataRepository {
```

Comment:
This is a matter of preference, but in my previous experience we weren't specifying `Network` / `Local` (just assuming that repository that uses local storage is called `Local`).
So I would name this class as `PaymentsDataRepository` and class it extends `IPaymentsDataRepository ` where `I` indicates that this class is just an interface and doesn't contain any logic.
What are your thought on this?

Reply:
I believe that naming classes smth like `ClassImpl` or `ClassInterface` is not the best idea. Implementation should give a hint on how it's working, ex. `Local/Network/Persistent`. Not sure that it's worth naming interfaces with `I`, considering that in dart it's still an abstract class. What's important - to give a name, which tells client what data/functionality he gets.